docs/source/api/plots.rst

```diff
@@ -7,17 +7,7 @@ Plots
 Plots are delegated to the
 `ArviZ <https://arviz-devs.github.io/arviz/index.html>`_.
 library, a general purpose library for
-"exploratory analysis of Bayesian models."
-For plots, ``pymc3.<function>`` are now aliases
-for ArviZ functions. Thus, the links below will redirect you to
-ArviZ docs:
+"exploratory analysis of Bayesian models".
 
-- :func:`pymc3.traceplot <arviz:arviz.plot_trace>`
-- :func:`pymc3.plot_posterior <arviz:arviz.plot_posterior>`
-- :func:`pymc3.forestplot <arviz:arviz.plot_forest>`
-- :func:`pymc3.compareplot <arviz:arviz.plot_compare>`
-- :func:`pymc3.autocorrplot <arviz:arviz.plot_autocorr>`
-- :func:`pymc3.energyplot <arviz:arviz.plot_energy>`
-- :func:`pymc3.kdeplot <arviz:arviz.plot_kde>`
-- :func:`pymc3.densityplot <arviz:arviz.plot_density>`
-- :func:`pymc3.pairplot <arviz:arviz.plot_pair>`
+Functions from the `arviz.plots` module are available through ``pymc3.<function>`` or ``pymc3.plots.<function>``,
+but for their API documentation please refer to the `ArviZ documentation <https://arviz-devs.github.io/arviz/api/plots.html>`_.
```

Comment:
```suggestion
but for their API documentation please refer to the :ref:`ArviZ documentation <arviz:plot_api>`.
```

It's probably a good idea to check it works, but we can use intersphinx for this too, we have a label to link to the plots section in api docs.

Reply:
Is it work the risk? Doesn't it just result in the same link to the original ArviZ docs?

Reply:
I can check it works locally, I think it is worth the risk because even if I recently reorganized the docs recently (chaning the urls) and I don't think we will modify that anytime soon, what we should do (hopefully) soon is getting multiple versions of the docs on the website which will probably alter the links by adding a `stable/` or `latest` somewhere. If we use intersphinx, regenerating the docs fixes everything (at most we have to edit the intersphinx mapping), if we don't we have to manually change all the links